rllib/agents/dqn/dqn_policy.py

```diff
@@ -260,7 +259,8 @@ def build_q_networks(policy, q_model, input_dict, obs_space, action_space,
                        config["model"])
     policy.output_actions, policy.action_prob = qvp.action, qvp.action_prob
 
-    return policy.output_actions, policy.action_prob
+    return (policy.output_actions, tf.log(policy.action_prob)
```

Comment:
can you also add docstring for this? otherwise users won't know what broke if they are importing/subclassing.

why all of this redundancy - why not just have a property that `self.action_logp = tf.log(self.action_prob)` in `policy`?